jetty-http/src/main/java/org/eclipse/jetty/http/HttpParser.java

```diff
@@ -1064,18 +1063,20 @@ else if (_endOfContent == EndOfContent.CHUNKED_CONTENT)
                 {
                     if (_fieldCache == null)
                     {
+                        // TODO should we bypass the builder in this case?
                         _fieldCache = (getHeaderCacheSize() > 0 && (_version != null && _version == HttpVersion.HTTP_1_1))
-                            ? new Index.Builder<HttpField>()
+                            ? new Index.Mutable.Builder<HttpField>()
                             .caseSensitive(false)
                             .mutable()
```

Comment:
This call isn't needed anymore.

Reply:
Unfortunately it is because `caseInsensitvie(false)` returns the wrong type of Builder.  I guess I could reorder to make that last.... or just not do the alternative constructor....

Reply:
Right... by exposing this builder's constructor you've opened a whole new range of potential maintenance problems. We have to seriously ponder if it's worth it.

Reply:
yep - I don't like it... it is a halfway house.  If the builder is not efficient enough, we should go directly to the impl, or have a very special purpose static method builder.... let me try those approaches....